thinc/backends/cblas.pxd

```diff
@@ -33,6 +34,8 @@ cdef class CBlas:
 cdef daxpy_ptr daxpy(CBlas cblas) nogil
 cdef saxpy_ptr saxpy(CBlas cblas) nogil
 cdef sgemm_ptr sgemm(CBlas cblas) nogil
+cdef sscalv_ptr sscalv(CBlas cblas) nogil
```

Comment:
I think this should be `sscal` right? If I am not mistaken `sscalv` is a BLIS symbol/name.

https://www.intel.com/content/www/us/en/develop/documentation/onemkl-developer-reference-c/top/blas-and-sparse-blas-routines/blas-routines/blas-level-1-routines-and-functions/cblas-scal.html

While at it, we should probably also add `dscal`.

Reply:
Ah, yes. Good catch! Will fix/add.